pkg/rtc/participant.go

```diff
@@ -684,20 +639,16 @@ func (p *ParticipantImpl) Close(sendLeave bool, reason types.ParticipantCloseRea
 		onClose(p, disallowedSubscriptions)
 	}
 
-	// Close peer connections without blocking participant close. If peer connections are gathering candidates
-	// Close will block.
-	go func() {
-		for _, dt := range downTracksToClose {
-			dt.CloseWithFlush(sendLeave)
-		}
-
-		p.TransportManager.Close()
-	}()
+	go p.TransportManager.Close()
```

Comment:
participant would only be concerned with Transport, downtracks would be closed by SubscriptionManager